workitem/workitemtype.go

```diff
@@ -35,6 +35,9 @@ const (
 	SystemArea                = "system.area"
 	SystemCodebase            = "system.codebase"
 	SystemLabels              = "system.labels"
+	SystemBoardcolumns        = "system.boardcolumns"
+
+	SystemBoard = "Board"
```

Comment:
every other constant value is in small letters so even this should be `SystemBoard` = "board"

Reply:
My question would be where this is used and if it is a field like the others.

Reply:
It is used in search.rearch_repository.go in determineLiteralType().